ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/processes/panel/ProcessesPanelView.java

```diff
@@ -141,6 +141,12 @@ void addWidget(
      */
     void onCloseCommandOutputClick(ProcessTreeNode node);
 
+    /**
+     * Is called when user is going to close command tab.
```

Comment:
the same as [here](https://github.com/eclipse/che/pull/6156#discussion_r137267549)